File: Find Pivot Index/find-pivot-index.cpp
class Solution {
public:
    int pivotIndex(vector<int>& nums) {
        int left = 0, total = 0;
        for (int i = 0; i < nums.size(); i++)
            total += nums[i];
        for (int i = 0; i < nums.size(); i++) {
            if (left == total - left - nums[i])
                return i;
            left += nums[i];
        }
        return -1;
    }
};

File: Find Pivot Index/find-pivot-index.py
class Solution(object):
    def pivotIndex(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        left = 0
        total = sum(nums)
        for i in range(len(nums)):
            if left == total - left - nums[i]:
                return i
            left += nums[i]
        return -1
